examples/dynamo/eg7/alg.x90

```diff
@@ -1,3 +1,14 @@
+! Modifications copyright (c) 2017, Science and Technology Facilities Council
+! -----------------------------------------------------------------------------
+! (c) The copyright relating to this work is owned jointly by the Crown,
+! Met Office and NERC 2014.
+! However, it has been created with the help of the GungHo Consortium,
+! whose members are identified at https://puma.nerc.ac.uk/trac/GungHo/wiki
+!-------------------------------------------------------------------------------
+! Modified by I Kavcic, Met Office.
+!
+!> @brief Tests columnwise assembled (CMA) operators.
```

Comment:
Strictly speaking this is just an example rather than a test.

Reply:
Changed "Tests" to "Example of".

Reply:
Corrected BSD license as agreed.